_tests_/testing.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Welcome from '../client/src/index.jsx';
+
+describe('Welcome (Snapshot)', () => {
+  it('Welcome renders hello world', () => {
+    const component = renderer.create(<Welcome />);
+    const json = component.toJSON();
+    expect(json).toMatchSnapshot();
+  });
+});
```

Comment:
@janicelam21 
@poema-cavalcanti 

If you're following the Airbnb style guide there should be a new line at the end of this file! (It may be worth checking if your ESLint is working).

Reply:
I'll fix the red squigglies!